tests/DisputeTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Stripe;
+
+class DisputeTest extends TestCase
+{
+    public function testUrls()
+    {
+        $this->assertSame(Dispute::classUrl(), '/v1/disputes');
+        $dispute = new Dispute('dp_123');
+        $this->assertSame($dispute->instanceUrl(), '/v1/disputes/dp_123');
+    }
+
+    private function createDisputedCharge()
+    {
+        $card = array(
+            'number' => '4000000000000259',
+            'exp_month' => 5,
+            'exp_year' => date('Y') + 1
+        );
+
+        $c = Charge::create(
+            array(
+                'amount' => 100,
+                'currency' => 'usd',
+                'card' => $card
+            )
+        );
+        $c = Charge::retrieve($c->id);
+
+        $attempts = 0;
+
+        while ($c->dispute === null) {
+            if ($attempts > 5) {
+                throw "Charge is taking too long to be disputed";
+            }
+            sleep(10);
```

Comment:
Given that a consumer is unlikely to be between 10 and 50 seconds behind (i.e. if it's 10s behind it's probably a few minutes behind), do we want to make this 1s?

Alternatively, we now have the infra to stub out responses in PHPUnit, so we could just stub out a dispute.